osgp/platform/osgp-adapter-domain-smartmetering/src/main/java/org/opensmartgridplatform/adapter/domain/smartmetering/infra/jms/ws/WebServiceResponseMessageSender.java

```diff
@@ -35,51 +32,47 @@
      * live.
      *
      * @param responseMessage
-     *            The response message to send.
+     *         The response message to send.
      * @param timeToLive
-     *            The custom time to live value in milliseconds.
+     *         The custom time to live value in milliseconds.
      */
     public void send(final ResponseMessage responseMessage, final Long timeToLive, final String messageType) {
 
         // Keep the original time to live from configuration.
-        final Long originalTimeToLive = this.jmsTemplate.getTimeToLive();
+        final long originalTimeToLive = this.jmsTemplate.getTimeToLive();
         if (timeToLive != null) {
             // Set the custom time to live.
             this.jmsTemplate.setTimeToLive(timeToLive);
         }
 
-        this.jmsTemplate.send(new MessageCreator() {
+        this.jmsTemplate.send(session -> {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this lambda expression number of lines from 24 to at most 20. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1188)